mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/TokenDetailScreen.tsx

```diff
@@ -414,7 +414,7 @@ function TokenSummary (props: { token: WalletToken, border?: boolean, usdAmount:
                   displayType='text'
                   renderText={(value) => (
                     <ThemedTextV2
-                      style={tailwind('flex-wrap mr-1 text-sm font-semibold-v2 text-right')}
+                      style={tailwind('flex-wrap mr-1 font-semibold-v2 text-right')}
```

Comment:
Not in the reported bug, but let's change this to match figma
```suggestion
                      style={tailwind('flex-wrap font-semibold-v2 text-right')}
```